metacat-main/src/main/java/com/netflix/metacat/main/services/search/ElasticSearchUtilImpl.java

```diff
@@ -118,7 +122,7 @@ public ElasticSearchUtilImpl(
     public void delete(final String type, final String id) {
         try {
             RETRY_ES_PUBLISH.call(() -> {
-                client.prepareDelete(esIndex, type, id).execute().actionGet();
+                client.prepareDelete(esIndex, type, id).execute().actionGet(esCallTimeout);
                 return null;
```

Comment:
Can we catch the exception due to timeout() , i.e. ElasticsearchTimeoutException() and metrics them separately? So that we know how the timeout configures perform.